src/ensembl/src/content/app/species-selector/components/assembly-selector/AssemblySelector.test.tsx

```diff
@@ -49,14 +50,29 @@ describe('<AssemblySelector />', () => {
 
     const selectElement = wrapper.find(Select);
     expect(selectElement.length).toBe(1);
-    expect(selectElement.prop('onSelect')).toBe(onSelect);
 
     const options: any = selectElement.prop('options');
     const selectedOption = options.find(
       ({ isSelected }: { isSelected: boolean }) => isSelected
     );
+    const selectedOptionIndex = options.findIndex(
+      ({ isSelected }: { isSelected: boolean }) => isSelected
+    );
 
     expect(options.length).toEqual(defaultProps.assemblies.length);
-    expect(selectedOption.value).toBe(defaultProps.genomeId);
+    expect(selectedOption.value).toBe(selectedOptionIndex);
```

Comment:
The description says that it should render a select if multiple assemblies are available, which is done on the line:
`expect(selectElement.length).toBe(1);`
Would it be better to add some comments about what is being checked on this line?

Reply:
Good point. Split the test into two to make it more obvious.